catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -1125,19 +1134,44 @@ public DeleteResponse delete(DeleteRequest deleteRequest)
 
         DeleteResponse deleteResponse = null;
         try {
+            List<Filter> idFilters = new ArrayList<>();
+            for (Serializable serializable : deleteRequest.getAttributeValues()) {
```

Comment:
Could also use a stream/map/collect here but I'm not sure it'd be any more concise.